lib/graph/__init__.py

```diff
@@ -52,12 +52,12 @@ def __init__(self, graph, id="", radius=8, style=style.DEFAULT, category="", lab
         self._betweenness = None
         self._eigenvalue = None
         
-        for k, v in properties.items():
+        for k, v in list(properties.items()):
```

Comment:
Looks like this one might not need wrapping in a list, as it doesn't seem to try and modify `properties` ?